docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -127,18 +157,20 @@ include::{docdir}/shared-ml-concepts.asciidoc[tag=feature-importance]
 [[dfa-classification-evaluation]]
 ==== Measuring model performance
 
-You can measure how well the model has performed on your dataset by using the 
+You can measure how well the model has performed on your data set by using the 
 `classification` evaluation type of the 
 {ref}/evaluate-dfanalytics.html[evaluate {dfanalytics} API]. The metric that the 
-evaluation provides you is the multi-class confusion matrix which tells you how 
-many times a given data point that belongs to a given class was classified 
-correctly and incorrectly. In other words, how many times your data point that 
-belongs to the X class was mistakenly classified as Y.
+evaluation provides you is a confusion matrix. The more classes you have, the 
+more complex the confusion matrix is. The matrix tells you how many data points 
+that belong to a given class were classified correctly and incorrectly. In other 
+words, how many data points that belong to the X class were classified correctly 
+and mistakenly classified as Y or Z or any other class in the data set other than 
+X.
```

Comment:
> In other words, how many data points...

In my opinion, this sentence is redundant (i.e. doesn't add anything to the previous statement). What I think would be more interesting to add (in a separate PR if necessary) is what to do about poor classification results.  Can we provide tips about whether to improve it by re-running with feature importance enabled or using the explain API to determine which fields we should explicitly include or exclude? Or by curating the training data?

Reply:
I deleted the sentence and address the rest in a separate PR.